colorfield/validators.py

```diff
@@ -17,3 +17,50 @@
     _("Enter a valid hexa color, eg. #00000000"),
     "invalid",
 )
+
+COLOR_RGB_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgb\("
+    # first number: red channel
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # second number: green channel
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # third number: blue channel
+    r"(\d{1,3})"
+    # closing parenthesis
+    r"\)$"
+)
+color_rgb_validator = RegexValidator(
+    COLOR_RGB_RE,
+    _("Enter a valid rgb color, eg. rgb(128, 128, 128)"),
+    "invalid",
+)
+COLOR_RGBA_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgba\("
+    # first number: red channel
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # second number: green channel
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # third number: blue channel
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # alpha channel: decimal number between 0 and 1
+    r"(0(\.\d{1,2})?|1)"
```

Comment:
Maybe accepting an optional `.0` after the `1` would be more flexible `r"(0(\.\d{1,2})?|1(\.0)?)"`

Reply:
Nice, I haven't thought of this use case. Will add now.